### PR TITLE
add ace arsenal button to main menu

### PR DESCRIPTION
**Pull request by commy2**
**When merged this pull request will:**
- [x] icons
- [x] enable debug console (requires: https://github.com/CBATeam/CBA_A3/pull/843)
- [x] replace vanilla arsenal with ace arsenal inside the mission
- [x] cleanup scripts, add features?
- [x] Esc to exit

### CODE REVIEW

File: addons/arsenal/stringtable.xml [hunk 1]
```diff
@@ -433,5 +433,17 @@
             <Italian>Impossibile aprire l'arsenale ACE</Italian>
             <Japanese>ACE 武器庫を開けません</Japanese>
         </Key>
+          <Key ID="STR_ACE_Arsenal_Mission">
+            <English>ACE Virtual Arsenal</English>
```
Comment (jonpas) [inline addons/arsenal/stringtable.xml]: Should be just "ACE Arsenal".

**Comment by jonpas** [comment]
> replace vanilla arsenal with ace arsenal inside the mission

Meaning in that mission that loads? As in a back to ACE Arsenal instead of Virtual Arsenal?

**Comment by commy2** [comment]
The ACE Arsenal mission loads the vanilla arsenal, both automatically and by returning from the mission.
This is because all scripts of the mission are copy pasted atm.

File: addons/arsenal/stringtable.xml (hunk 1 above)
Comment (commy2) [reply addons/arsenal/stringtable.xml]: :+1: The descriptions are placeholder / copy paste atm.

**Comment by Dystopian** [comment]
partly related to #5932. If setting is set there will be 2 ACE Arsenal in main menu :)

**Comment by bux** [comment]
Currently you can't exit the mission 😄

**Comment by commy2** [comment]
I know. lol

**Comment by alganthe** [comment]
I am alganthe, and this is my favorite PR in the ACE3 repo.

**Comment by bux** [comment]
When starting the mission invisble dudes shout Farsi at you. It's a bit ... disturbing.

File: addons/arsenal/stringtable.xml [hunk 2]
```diff
@@ -433,5 +433,19 @@
...
+          <Key ID="STR_ACE_Arsenal_Mission">
+            <English>ACE Arsenal</English>
+            <German>ACE-Arsenal</German>
+          </Key>
+          <Key ID="STR_ACE_Arsenal_ReturnToArsenal">
+            <English>Return to ACE Arsenal.</English>
+            <German>Zurück zum ACE-Arsenal.</German>
+          </Key>
+          <Key ID="STR_ACE_Arsenal_Mission_tooltip">
+            <English>Not as bad as the regular one.</English>
```
Comment (bux) [inline addons/arsenal/stringtable.xml]: Let's find something better than that.

**Comment by bux** [comment]
VA has one more button at the bottom: "Exit" and our "Close" is called "Try" in their UI.

Ours is a bit confusing

File: addons/arsenal/stringtable.xml (hunk 2 above)
Comment (commy2) [reply addons/arsenal/stringtable.xml]: I'm all ears.

Comment (dedmen) [reply addons/arsenal/stringtable.xml]: `A more splendid Arsenal experience`

Comment (jonpas) [reply addons/arsenal/stringtable.xml]: No, we aren't using "splendid", we aren't BI!

**Comment by commy2** [comment]
I'd like if it saves the last loadout you had when you exit and to restore it when you enter it again.

**Comment by commy2** [comment]
Done, done and done.

**Pull request merged**